modin/config/pubsub.py

```diff
@@ -278,6 +352,10 @@ def put(cls, value: Any) -> None:
         value : Any
             Config value to set.
         """
+        if cls._deprecation_descriptor is not None:
```

Comment:
Won't we get an attribute error for the parameters that do not have _deprecation_descriptor?

Reply:
[every `Parameter` has this field](https://github.com/dchigarev/modin/blob/2836ed3de380c299e68196cd7af9dafb7e5652bc/modin/config/pubsub.py#L216), so it shouldn't happen